tests/epyccel/test_arrays_multiple_assignments.py

```diff
@@ -165,6 +166,59 @@ def f(c):
     c = np.random.random()
     assert f(c) == g(c)
 
+#==============================================================================
+def test_Reassign_to_Target(language):
```

Comment:
This tests whether an error is raised in the semantic stage. This means that the codegen stage is never reached, so the language argument is pointless

Reply:
will change it